test/integration/taxon_browsing_test.rb

```diff
@@ -233,6 +247,16 @@ def then_i_can_see_the_meta_description
     )
   end
 
+  def then_page_has_meta_robots
+    content = page.find('meta[name="robots"]', visible: false)['content']
+
+    assert_equal(
+      "noindex, nofollow",
+      content,
+      "The content of the meta description should be the taxon description"
```

Comment:
Minor thing. The content of the optional failure message doesn't match what's being tested. You might want to change to something like:

'The content of the robots meta tag should be "noindex, nofollow"'

Reply:
Oh nice catch!! 
This is why you don't copy and paste, kids!!